### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by msbranin**
Updated themes that had the wrong info.

### CODE REVIEW

**Comment by msbranin** [comment]
We stall have themes like athlete that Allow you to choose a +1 to str, dex or con... What would be the best way to handle those? Either the Ability boost needs to be modifiable or we could create 3 different Athlete themes, each with a different + 1ability boost.

File: src/items/themes/giantblood.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "type": "",
     "modifiers": [],
     "abilityMod": {
-      "ability": "str",
```
Comment (CptTwinkie) [inline src/items/themes/giantblood.json:14]: Maybe there's something I'm not understanding here but the old value looks correct to me
image

Comment (msbranin) [reply src/items/themes/giantblood.json:14]: Whoops That was a mistake. Can you look over merc I know it was comepletely wrong in the original file as well as priest had the wrong stat mod

Comment (msbranin) [reply src/items/themes/giantblood.json:14]: Wait. In Foundry when you look at the Giantblood theme is Says +1 Con which is where the initial mistake came from.
Giantblood

**Review by CptTwinkie — APPROVED** [review]

**Comment by CptTwinkie** [comment]
This still needs a cook but the json seems to be correct now

**Comment by TimToxopeus** [comment]
Looks fine, I'll merge it and cook it myself. Thanks!

**Pull request merged**